geode-protobuf/src/main/proto/authentication_API.proto

```diff
@@ -17,8 +17,7 @@ syntax = "proto3";
 package org.apache.geode.internal.protocol.protobuf;
 
 message SimpleAuthenticationRequest {
```

Comment:
This message is not simple authentication specific anymore. This message should maybe be called `PropertyBasedAuthenticationRequest`.

Reply:
Going with the very simple AuthenticationRequest